docs/src/index.md

```diff
@@ -14,70 +14,46 @@ Whenever one of your package's dependencies releases a new breaking version, Com
-
-If none of those situations apply to you, then you don't need to set up an SSH deploy key for CompatHelper.
-
-### Instructions for setting up the SSH deploy key
-
-It is easy to set up an SSH deploy key for CompatHelper. Here are the instructions:
-1. `ssh-keygen -m PEM -N "" -f compathelper_key`
-2. `cat compathelper_key # This is the private key in PEM format. Copy this to your clipboard.`
-3. Go to the GitHub page for your package's repository, click on the **Settings** tab, then click on **Secrets**, and then click on **New repository secret**. Name the secret `COMPATHELPER_PRIV`. For the contents, paste in the PEM-formatted private key that you copied in the previous step. Make sure that you include the entire PEM-formatted private key, including the lines at the beginning and end that start with `-----BEGIN` and `-----END` respectively.
-4. `cat compathelper_key.pub # This is the public key. Copy this to your clipboard.`
-5. Go to the GitHub page for your package's repository, click on the **Settings** tab, then click on **Deploy keys**, and then click on **Add deploy key**. Name the deploy key `COMPATHELPER_PUB`. For the contents, paste in the public key that you copied in the previous step. Make sure that you give the key **write access**.
-6. `rm -f compathelper_key compathelper_key.pub`
-
-### Advanced notes
-
-When you supply the private key, you can either provide the raw PEM-formatted private key itself (as we did above), or you can provide the Base64-encoded form of the PEM-formatted private key.
-
-For example, if you already have a Base64-encoded PEM-formatted private key saved as a secret, you can re-use it. If e.g. the secret is named `DOCUMENTER_KEY`, then simply replace the line that looks like this:
-```yaml
-COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}
-```
-
-with this line:
-```yaml
-COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
+name: CompatHelper
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Everyday at midnight
+  workflow_dispatch:
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pkg.add("CompatHelper")
+        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
+      - name: CompatHelper.main()
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}
+        run: julia -e 'using CompatHelper; CompatHelper.main()'
 ```
+### Creating SSH Key
+If you use GitHub Actions to either test your packge using continuous integration, or build and deploy documentation you will need to create an SSH deploy key. If you wish to reuse an existing SSH key simplify modify the workflow above environment variable to use `COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}`. Otherwise follow the below instructions to generate a new key:
+
+1. Generate a new SSH key
```

Comment:
might be good to explain/give and example of making a base-64 encoded key. I know on GitLab at least, it makes more sense to do that, not sure it really matters with github

Reply:
This is taken from the existing documentation. Is there really a point to storing as base64?

Reply:
at least in GitLab, you can't hide the ssh key from being outputted in it's entirety (if it is output somehow), if you store it in base 64 then it can be masked during logging and just show up as `***`. 

The existing documentation I believe also mentions base-64 encoded ssh keys, but doesn't really explain it